Dfe.Academies.Academisation.IData/Establishment/Establishment.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Dfe.Academies.Academisation.IData.Establishment
+{
+	public class Establishment
+	{
+		public Establishment()
+		{
+			Gor = new Region();
```

Comment:
Do we want to use the name Gor? If so what does it mean?

Reply:
If it's to match a property, I think its probably fine, more curiosity as to where it comes from.